base/applications/taskmgr/graphctl.c

```diff
@@ -407,6 +419,52 @@ void GraphCtrl_Paint(TGraphCtrl* this, HWND hWnd, HDC dc)
     DeleteDC(memDC);
 }
 
+void GraphCtrl_ShiftGrid(TGraphCtrl* this)
+{
+    RECT rectCleanUp;
+    HPEN oldPen;
+    HPEN solidPen = CreatePen(PS_SOLID, 0, this->m_crGridColor);
+    int i;
+
+    if (this->m_dcGrid != NULL)
+    {
```

Comment:
```suggestion
    if (this->m_dcGrid == NULL)
        return;
```

Reply:
But then he would need to call the `DeleteObject(solidPen);` there too.

Reply:
I changed it. Is it good now?

Reply:
> But then

Obviously.

> I changed it. Is it good now?

Yes.